src/lbaf/Execution/lbsInformAndTransferAlgorithm.py

```diff
@@ -189,6 +190,8 @@ def execute(self, p_id: int, phases: list, distributions: dict, statistics: dict
         for i in range(self.__n_iterations):
             self._logger.info(f"Starting iteration {i + 1} with total work of {total_work}")
 
+            start_time = time.time()
```

Comment:
Please add comment if you are going to skip a line (i.e. start a new logical block); otherwise do not add a new line

Reply:
Fixed, thanks!